### ISSUE TITLE
ObjectPage: RTL, accessibility and responsiveness issues

### ISSUE THREAD

**Issue by Neeeko** [SAP/ui5-webcomponents-react#345]
**Describe the bug**
I found a couple of issues in the ObjectPage.

1. RTL
The navigation bar doesn't support RTL well. Adding `dir="rtl"` crushes the Navigation Bar on the right side of the page. (see Screenshots 1 and 2)

2. Accessibility
The navigation bar isn't accessibility-friendly. Contrary to the UI5-WebComponents [Tab Container](https://sap.github.io/ui5-webcomponents/playground/components/TabContainer/), we cannot navigate using tabs/arrows/space. There's no dotted outline. (see Screenshots 3 to see how it's handled in UI5-WebComponents [Tab Container](https://sap.github.io/ui5-webcomponents/playground/components/TabContainer/))

3. Responsiveness
On mobile devices, both the Header (even the not custom stuff such as the title and subtitle) and the Navigation bar aren't fully responsive. (see Screenshot 4)
UI5-WebComponents [Tab Container](https://sap.github.io/ui5-webcomponents/playground/components/TabContainer/) has some nice buttons or horizontal scrolling. (see Screenshot 5)

**To Reproduce**
See this codesandbox: https://codesandbox.io/s/webcomponents-react-objectpage-fy257
1. Steps to reproduce the RTL issue:
Add `dir="rtl"` to your root div.

2. Steps to reproduce the accessibility issue:
Try to navigate through the UI5-WebComponents [Tab Container](https://sap.github.io/ui5-webcomponents/playground/components/TabContainer/). You should be able to navigate thanks to the arrows and space keys.
You won't be able to do the same for the ObjectPage Navigation Bar (see the [codesandbox](https://codesandbox.io/s/webcomponents-react-objectpage-fy257)).

3. Steps to reproduce responsiveness issues:
Open Google Chrome Dev Tools then toggle the Device Toolbar. 
Reduce the size to "Mobile L".

**Expected behavior**
1. Setting the RTL mode should render an ObjectPage as readable as the LTR one. The user experience shouldn't be impacted.

2. All the default non-customizable elements of the ObjectPage should be accessible: Readable by Screen Reader and should allow keyboard navigation.

3. The ObjectPage elements should be properly displayed regardless of the viewport size.

**Screenshots**
![Screen Shot 1](https://user-images.githubusercontent.com/7316793/76333922-a2228380-62f2-11ea-8282-52620026f7b3.png)
_Screenshot 1_

![Screen Shot 2](https://user-images.githubusercontent.com/7316793/76333924-a2bb1a00-62f2-11ea-8c22-92fe30865d83.png)
_Screenshot 2_

![Screen Shot 3](https://user-images.githubusercontent.com/7316793/76333991-bfefe880-62f2-11ea-9c0c-207feac13db7.png)
_Screenshot 3_

![Screen Shot 4](https://user-images.githubusercontent.com/7316793/76334013-c5e5c980-62f2-11ea-88d3-f06f6684631e.png)
_Screenshot 4_

![Screen Shot 5](https://user-images.githubusercontent.com/7316793/76406068-74d4e480-6389-11ea-9c1a-fe9bfbc0eddc.png)
_Screenshot 5_

**UI5 Web Components for React Information**
Operating System:
`@ui5/webcomponents-react` version: 0.8.6
Do you have a dependency to`@ui5/webcomponents` in your package.json? If yes, which version?: 1.0.0-rc.5

Thanks! :)

**Comment by MarcusNotheis** [comment]
Hi @Neeeko 
thanks for submitting this issue. I just released a new release candidate of the library which contains a full refactoring of the ObjectPage. In this refactoring I replaced our custom navigation bar with the UI5 Tab Container Web Component, so Issue 1 and 2 should be way better now.
If you have any feedback or experience new bugs please let me know.

For the responsiveness-issue: We are still working on it.

Release Candidate: `@ui5/webcomponents-react@0.8.7-rc.0`

**Comment by Neeeko** [comment]
Hi @MarcusNotheis 
Thanks for this great work which indeed fixes the problems that I listed.

> If you have any feedback or experience new bugs please let me know.
For the responsiveness-issue: We are still working on it.

The only thing that I noticed is that when you reduce the size of the browser viewport, the ObjectPageSection grows to the bottom infinitely.

Do you want me to close this issue and raise another one?

**Comment by MarcusNotheis** [comment]
Hi @Neeeko thank you :)
Yes, please feel free to close close this issue and transfer the missing header responsiveness as well as the growth issue into new issues? Thanks for your help!
